sortinghat/core/schema.py

```diff
@@ -605,6 +648,28 @@ def resolve_individuals(self, info, filters=None,
                                                              page,
                                                              page_size=page_size)
 
+    @check_auth
+    def resolve_job(self, info, job_id):
+        job = find_job(job_id)
+
+        status = job.get_status()
+        result = job.result
+
+        if job.result:
+            errors = result['errors']
+            affiliated = [
+                AffiliationResultType(uuid=uuid, organizations=orgs)
+                for uuid, orgs in result['results'].items()
+            ]
+        else:
+            errors = None
+            affiliated = None
+
+        return JobType(job_id=job_id,
```

Comment:
Although paginating jobs does not make sense here (as we are asking for one job), would it be possible to paginate the `AffilitationResultType` results somehow?

Reply:
It would be possible but we have to think how. The results are stored in Redis all in the same structure which means we will have to do the pagination once the results are gathered by the django view. I think we can leave it this way not and think about possible solutions, if needed, for the future.